README.md

```diff
@@ -1,15 +1,103 @@
-# MAX Image Caption Generator Web App
+# Generating captions for images and filtering based on image content
```

Comment:
Though this describes the use case for the web app well I don't think it's the best description for the code pattern. I think something between this and my original title `Create a web app to interact with machine learning generated image captions` (which is a bit boring) would be better. Maybe:

> Create a web app to filter images based on deep learning generated captions

or something along those lines. I think keeping the "create a web app" part is important but your title describes the rest better. @stevemart whats your opinion based on previous code patterns? and @frreiss what's your opinion based on presenting the demo?

Reply:
@ajbozarth use the part that you want to highlight first, so machine learning, right?

> Use a machine learning model to filter images in a web app